labrad/node/__init__.py

```diff
@@ -68,39 +68,42 @@
 
 from __future__ import with_statement
 
+from ConfigParser import SafeConfigParser
+from datetime import datetime
+import logging
+import logging.handlers
 import os
-import sys
+import shlex
 import socket
-from datetime import datetime
-from ConfigParser import SafeConfigParser
 import StringIO
+import sys
 import zipfile
-import logging
-import logging.handlers
 
-import labrad
-from labrad import util, types as T, constants as C
-from labrad.server import ILabradServer, LabradServer, setting
-from labrad.util import dispatcher, findEnvironmentVars, interpEnvironmentVars
-import labrad.support
-
-from twisted.application.service import IService, MultiService
+from twisted.application.service import MultiService
 from twisted.application.internet import TCPClient
 from twisted.internet import defer, reactor
 from twisted.internet.defer import inlineCallbacks, returnValue
 from twisted.internet.protocol import ProcessProtocol
 from twisted.internet.error import ProcessDone, ProcessTerminated
-from twisted.python import log, failure, usage
+from twisted.python import usage
 from twisted.python.runtime import platformType
-from twisted.python.components import registerAdapter
 from twisted.plugin import getPlugins
 from zope.interface import Interface, implements
 
+import labrad
+from labrad import util, types as T, constants as C
+from labrad.server import ILabradServer, LabradServer, setting
+import labrad.support
+from labrad.util import dispatcher, findEnvironmentVars, interpEnvironmentVars
+
+
 LOG_LENGTH = 1000 # maximum number of lines of stdout to keep per server
 
+
 class IServerProcess(Interface):
     pass
 
+
 class ServerProcess(ProcessProtocol):
     """A class to represent a running server instance."""
```

Comment:
I won't ask for a full doc string, but maybe just the attribute(s) added by this PR?